doc/examples/fifoInterfaceExample.cpp

```diff
@@ -1,141 +1,133 @@
 /* *****************************************************************************
  * A.L.E (Arcade Learning Environment)
  * Copyright (c) 2009-2013 by Yavar Naddaf, Joel Veness, Marc G. Bellemare,
- *  Matthew Hausknecht, and the Reinforcement Learning and Artificial Intelligence 
- *  Laboratory
- * Released under the GNU General Public License; see License.txt for details. 
+ *  Matthew Hausknecht, and the Reinforcement Learning and Artificial
+ *Intelligence Laboratory Released under the GNU General Public License; see
```

Comment:
Is the removal of the newline intentional here? the first line lists the ALE authors, the second is about the license.

Reply:
That's totally bogus! Hang on, I'll review everything.